scripts/make_fake_pm_data.js

```diff
@@ -0,0 +1,134 @@
+const async = require("async");
+const fs = require("fs");
+const _ = require("underscore");
+const json2csv = require("json2csv");
+const argv = process.argv.slice(2);
+const BUILDINGS = require("./" + argv[0]);
+
+function randomNumber(min, max) {
+    return Math.floor(Math.random() * (max - min + 1) + min);
+}
+
+function randomNumberOfStandardDeviations(min, max) {
+    return ((Math.random() * (max - min + 1) + min))
+}
+
+function shiftCoordinates(coordinate, shiftValue) {
+    return (coordinate - shiftValue);
+}
+
+function standardDeviation(mean, dataArray, numberOfEntries) {
+    let sumDistanceFromMean = 0 
+    dataArray.map(data => {
+        sumDistanceFromMean += Math.pow((data - mean), 2);
+    })
+
+    let stdDev = Math.round(Math.sqrt((sumDistanceFromMean / numberOfEntries)))
+    return stdDev;
+}
+
+function initialRead(cb) {
+    console.log("Reading data...")
+    const buildingJSON = BUILDINGS.map(building => {
+        return building
```

Comment:
I don't think this `map` is doing anything. You could just say `cb(null, BUILDINGS)` and it'd be the same. You already read the JSON by requiring it above.